frontend/src/pages/home/modelCatalog/ModelCatalogSectionFooter.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Button, Content, Flex, FlexItem } from '@patternfly/react-core';
+import { useNavigate } from 'react-router-dom';
+
+const ModelCatalogSectionFooter: React.FC<{
+  shownModelCount: number;
+  totalModelCount: number;
+}> = ({ shownModelCount, totalModelCount }) => {
+  const navigate = useNavigate();
+
+  return (
+    <Flex
+      gap={{ default: 'gapMd' }}
+      alignItems={{ default: 'alignItemsCenter' }}
+      data-testid="model-catalog-footer"
+    >
+      <FlexItem>
+        {shownModelCount ? (
+          <Content>
+            <Content component="small">
+              {shownModelCount < totalModelCount
+                ? `Showing ${shownModelCount} of all models`
+                : 'Showing all models'}
+            </Content>
+          </Content>
+        ) : null}
+      </FlexItem>
+      <FlexItem>
+        <Button
```

Comment:
Use `Link` from `react-router-dom` here instead of `Button` component to make it an anchor. The link button should only be used when we want to use the link styles but want it to be a `<button>` instead of an anchor.